Header header

float64 TimeStamp

float64 Front_Right_Track_Angle
float64 Front_Left_Track_Angle
float64 Rear_Right_Track_Angle
float64 Rear_Left_Track_Angle
float64 Rear_Bogie_Angle
